data/quests.ts

```diff
@@ -1,6 +1,109 @@
 import { ListType } from "./types";
 
 export const Quests: Array<ListType> = [
+  /* TEMPLATE     
+  {
+    // Can be left blank to be filled later
+    id: "",
+
+    // Name of the NPC/ending
+    name: "",
+
+    requirements: [
+      {
+        // Can be left blank to be filled later
+        id: "",
+
+        // Short sentence in brief describing necessary actions with few details
+        // Acceptable details:
+        // - Nearby Site of Grace, or Map Section
+        // - Necessary bosses and items
+        // - Dialog options
+        description: ""
+      },
+    ]
+  },
+  */
+  {
+    id: "",
+    name: "Kenneth Haight (Incomplete)",
```

Comment:
I've added what information is currently known about this questline. So I'd leave this without the incomplete line. If anything else is found about this, then it can be added later.

```suggestion
    id: "",
    name: "Kenneth Haight",
```